example/docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+version: '3.3'
+
+services:
+
+  filebeat:
+    image: docker.elastic.co/beats/filebeat:$ELK_VERSION
+    volumes:
+      ## Make sure to set this to the directory where your lancache logs are stored.
+      - ${CACHE_ROOT}/logs:/data/logs
+      - ./filebeat/filebeat.yml:/usr/share/filebeat/filebeat.yml
+    depends_on:
+      - logstash
+
+  ## ELK Stack
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:$ELK_VERSION
+    volumes:
+      - type: bind
+        source: ./elasticsearch/elasticsearch.yml
+        target: /usr/share/elasticsearch/config/elasticsearch.yml
+        read_only: true
+      - type: volume
+        source: elasticsearch
+        target: /usr/share/elasticsearch/data
+    ports:
+      - "9200:9200"
+    environment:
+      ES_JAVA_OPTS: ${ES_JAVA_OPTS}
+      ELASTIC_PASSWORD: ${ELASTIC_PASSWORD}
+      discovery.type: single-node
+
+  logstash:
+    image: docker.elastic.co/logstash/logstash:$ELK_VERSION
+    volumes:
+      - type: bind
+        source: ./logstash/logstash.yml
+        target: /usr/share/logstash/config/logstash.yml
+        read_only: true
+      - type: bind
+        source: ../logstash.conf
+        target: /usr/share/logstash/pipeline/logstash.conf
+        read_only: true
+    environment:
+      LS_JAVA_OPTS: "-Xmx256m -Xms256m"
+      ELASTIC_PASSWORD: ${ELASTIC_PASSWORD}
+    depends_on:
+      - elasticsearch
+
+  kibana:
+    image: docker.elastic.co/kibana/kibana:$ELK_VERSION
+    volumes:
+      - type: bind
+        source: ./kibana/kibana.yml
+        target: /usr/share/kibana/config/kibana.yml
+        read_only: true
+    environment:
+      ELASTIC_PASSWORD: ${ELASTIC_PASSWORD}
+    ports:
+      - "5601:5601"
+    depends_on:
+      - elasticsearch
+  dashboard-importer:
+    image: appropriate/curl
+    environment:
+      ELASTIC_PASSWORD: ${ELASTIC_PASSWORD}
+    volumes:
+      - type: bind
+        source: ../everything.ndjson
+        target: /usr/share/tmp/everything.ndjson
+        read_only: true
+    command: sh -c "/usr/bin/curl -X POST http://192.168.91.8:5601/api/saved_objects/_import -u elastic:$ELASTIC_PASSWORD -H kbn-xsrf:true -F 'file=@/usr/share/tmp/everything.ndjson'"
```

Comment:
Looks like a hardcoded ip address from somewhere :D think we can do http://elasticsearch:5601/

Reply:
Doh, fixed.